maud/Cargo.toml

```diff
@@ -27,6 +27,7 @@ actix-web-dep = { package = "actix-web", version = "4", optional = true, default
 tide = { version = "0.16.0", optional = true, default-features = false }
 axum-core = { version = "0.4", optional = true }
 http = { version = "1", optional = true }
```

Comment:
Hmm, looks like Axum uses http 1.0, but Warp uses http 0.2. But since the Warp integration doesn't use the http dependency directly, it still works out? Still something to look out for in the future.

Reply:
Noted, I'll have a close eye on it. 🙇